.github/ISSUE_TEMPLATE/4-chore.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Chore
+about: Work on a chore needing completion
```

Comment:
I think this should be more suggestive since this description will appear when a new issue is open. Maybe something like: `Create an issue for a chore needing completion`

Reply:
Agreed. Updated.